js-section/script.js

```diff
@@ -0,0 +1,71 @@
+// Like Heart code
+const heart = document.querySelector(".heart");
+let heartActive = false;
+
+heart.addEventListener('click', () => {
+  heartActive = !heartActive;
+  if (heartActive) {
+    heart.classList.add('bg-red');
+  } else {
+    heart.classList.remove('bg-red');
+  }
+});
+// End
+
+const plusButton = document.querySelector(".sum");
+const minusButton = document.querySelector(".minus");
+const quantityNumber = document.querySelector(".number");
+const totalSum = document.getElementById("total");
+
+let quantity = 0;
+
+const aditionalsList = [{name: "Pan de Pita relleno - Mini", price: 25, on:false}, {name: "Pan de Pita relleno - Grande", price: 50, on:false}, {name: "Agregar logo de tu empresa", price: "Sin costo", on:false}, {name: "Agregar una tarjeta con mensaje", price: "Sin costo", on:false}];
+
+const calculateTotalSum = () => {
+  let total = 1580;
+  aditionalsList.forEach((e) => {
+    if (e.on && e.price > 0) {
+      total += e.price;
+    }
+  });
+  const multipliedPrice = total * quantity;
+  totalSum.innerHTML = multipliedPrice;
+}
+
+aditionalsList.forEach((e, index) => {
+  const html = `<div class="flex gap-3">
+              <label class="switch">
+                <input type="checkbox" id="extra${index}">
+                <span class="slider round"></span>
+              </label>
+              <div class="flex flex-col gap-0-5">
+                <p class="text-base leading-tight tracking-tight font-normal"> ${e.name} </p>
+                <p class="text-sm font-light leading-snug tracking-tight">$${e.price} adicional</p>
+              </div>
+            </div>`
+  const container = document.getElementById("extras");
+  container.innerHTML += html;
+});
```

Comment:
Same abbreviation issue here. Maybe we can use the appendChile function and achieve the same result. Also, the getElementById can be moved out of the forEach